<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>481</width>
     <height>391</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>1</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>120</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入要插入的节点文本</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addTopLevelButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>190</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>添加到顶层元素</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addSelectedButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>250</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>添加到选中元素</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeSelectedButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>310</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>删除选中元素</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
